### ISSUE TITLE
No route can be found when traveling to Viena 

### ISSUE THREAD

**Issue by jehamannco** [valhalla/valhalla#4078]
Hello, we just found a strange bug, when trying to travel with destination Vienna (Austria). We get the following error message:

{"error_code":442,"error":"No path could be found for input","status_code":400,"status":"Bad Request"}

You can try to reproduce the bug here:
https://valhalla.openstreetmap.de/directions?profile=car&wps=13.7381437%2C51.0493286%2C16.37393116950989%2C48.20815528721601

If you slightly change the destination location like here:
https://valhalla.openstreetmap.de/directions?profile=car&wps=13.7381437%2C51.0493286%2C16.373394727706913%2C48.20794436239797

Then a route is found. Any ideas what the problem is?
Thanks!

**Comment by nilsnolde** [comment]
The problem is that loki can't find any car-accessible road close by. It's an unlucky coincidence, dropping a location right inside a pedestrian zone. It'd succeed if you could put a `"radius": 30` to the request. Custom `radius` is not enabled yet in the UI, but we will do that very likely soon. Even so, we could just put a `radius` by default to every location.

**Comment by kevinkreiser** [comment]
when the radius is 0 it will find the closest car accessible road. unless it's greater than 30km away loki should find something

**Comment by nilsnolde** [comment]
It does work with 30 m radius though. So when it’s 0 it’ll stop at the first eligible road it finds, but when it’s 30 it’ll correlate all roads within 30 m?

That would make sense if there’s some disconnected island, right.. didn’t debug any further either though

**Comment by kevinkreiser** [comment]
when its 0 (default) it will find the closest edge that fulfills the rest of the criteria, which by default have a reachability of 50 or 100 i think. when its 30 it will find ALL edges that fulfill the rest of the criteria. what im guessing is that the closest edge has a good enough reach but is still technically an island where as some other edge within 30m is connected to the larger network. to test this, instead of changing radius, i would change the reachability to something higher and see what it does to rule out this possibility

**Comment by jehamannco** [comment]
Hi, thanks for the quick reply. 

I have tried by increasing the radius as you guys suggested, and it works for us, but only with values above 40m. 

I have also tried to increase the minimum_reachability, but it does not succeed to find a path, not even with values as high as 2000.

For our application we would like to avoid such problems to happen, do you guys think it will be OK to fix this radius value, or it will be something not recommended doing?

**Comment by nilsnolde** [comment]
Yeah, just tried it myself, `minimum_reachability` has no effect in this case. Which leaves the question why this happens.

40 m radius is still fine IMO. Just be aware that you can get location-correlated edges which actually are 40 m away from the user input, if that's the cheapest path.

**Comment by nilsnolde** [comment]
yes: https://valhalla.github.io/valhalla/api/matrix/api-reference/#source-and-target-parameters

**Comment by kevinkreiser** [comment]
the service caps minimum reachability silently. so it probably maxes at 100 not 2000. you van change this in your valhalla.json server config. if that still doesn't fix it with a high min reach then we need to label this as a bug and find the real issue and fix it.

**Comment by jehamannco** [comment]
I tested this morning by changing the minimum_reachability to 100 in the valhalla.json config, and it still does not find a path. Only by setting radius to 40 works.
